src/main/java/carpet/script/language/Operators.java

```diff
@@ -202,6 +226,46 @@ public static void apply(Expression expression)
         });
         expression.addFunctionalEquivalence("<=", "nondecreasing");
 
+        expression.addMathematicalBinaryIntFunction("bitwise_shift_left", (l1, l2) -> {
```

Comment:
just use names (num, amount)

Reply:
k, but cant do that for rolling, cos I need to do mod 64